modules/layers/src/solid-polygon-layer/polygon.js

```diff
@@ -165,7 +196,8 @@ export function normalize(polygon, positionSize) {
           srcPositions,
           positionSize,
           srcHoleIndices[i - 1],
-          srcHoleIndices[i]
+          srcHoleIndices[i],
+          i === 0 ? WINDING.CLOCKWISE : WINDING.COUNTER_CLOCKWISE
```

Comment:
> A linear ring MUST follow the right-hand rule with respect to the area it bounds, i.e., exterior rings are counterclockwise, and holes are clockwise.

https://tools.ietf.org/html/rfc7946#section-3.1.6

Reply:
I suppose that GeoJSON input data must be organized that way, but here are no rules for how we organize data internally for optimal rendering?

@igorDykhta You indicated that following the GeoJSON standard leads to non-optimal results?

Reply:
I just selected WINDING.CLOCKWISE as front as it meant fewer changes on our side. There is no difference between cw & ccw. CCW is the default front in OpenGL and geojson, CW is considered the default by ESRI & Shapefile. 
I'll swap CW <> CCW, and add a note to the upgrade guide.

Reply:
It matters because our calculation of normals (and inherently lighting) calculation is dependent on this order. You need to verify that lighting is correct with this change.

Reply:
Looks like the lighting is consistent when CW is used for outer polygon, CCW for hole polygons.
![outer-as-CCW](https://user-images.githubusercontent.com/8210766/105349701-c62d4180-5bf2-11eb-9b69-c65623bb7bfb.png)
![outer-as-CW](https://user-images.githubusercontent.com/8210766/105349709-c7f70500-5bf2-11eb-8b8a-ee51066bef9c.png)